src/aiohomeconnect/model/event.py

```diff
@@ -21,14 +22,37 @@ class Event(DataClassJSONMixin):
     """Represent Event."""
 
     key: EventKey
-    name: str | None
-    uri: str | None
     timestamp: int
     level: str
     handling: str
-    value: str | float | bool
-    display_value: str | None = field(metadata=field_options(alias="displayvalue"))
-    unit: str | None
+    value: str | int | float | bool
+    ha_id: str | None = field(default=None, metadata=field_options(alias="haId"))
```

Comment:
When is `ha_id` not present?

Reply:
If the `id` attribute always has the ha_id, I think it's better to use that attribute to set `ha_id`.

Reply:
> When is `ha_id` not present?

When using the simulator API

> If the `id` attribute always has the ha_id, I think it's better to use that attribute to set `ha_id`.

Maybe we can delete the `Event` `ha_id` attribute and set the `EventMessage` `id` property to `ha_id`, as we know based on the experience that is it always the Home Appliance ID.

Reply:
Sounds good!

Reply:
We can raise an exception if it's different. We haven't added any library exceptions yet. That's still on the to do list. There's no error handling at all yet.